docker/service/src/main/java/uk/nhs/adaptors/scr/exceptions/NHSCodings.java

```diff
@@ -1,60 +0,0 @@
-package uk.nhs.adaptors.scr.exceptions;
-
-import org.hl7.fhir.r4.model.Coding;
-
-public enum NHSCodings {
```

Comment:
Why is this no longer needed/used?

Reply:
This has changed due to alignment to ACS setPermission error response designed by IOPS. See at bottom: https://simplifier.net/guide/SummaryCareRecordWithCodedData/SetPermission